network-api/src/NetworkAPI/Node/NetworkService.hs

```diff
@@ -1,7 +1,7 @@
-module NetworkAPI.Node.NodeClient
+module NetworkAPI.Node.NetworkService
```

Comment:
NetworkService should be abstracted from Node, thus no reason to put it into NetworkAPI.Node package.

Reply:
Also, please name modules consistently. E.g `module SubmitAPI.Service`, `module NetworkAPI.Service`